website/content/api-docs/catalog.mdx

```diff
@@ -408,6 +408,47 @@ The table below shows this endpoint's support for
   `X-Consul-Namespace` header. If not provided, the namespace will be inherited
   from the request's ACL token or will default to the `default` namespace. Added in Consul 1.7.0.
 
```

Comment:
Should the `node_meta` argument be marked as deprecated?

Reply:
Ya, I think this would be a good time to deprecate `node_meta`.